### PR TITLE
Add the monitoring of the "alpha.istio.io/canonical-serviceaccounts" annotation

### PR DESCRIPTION
**Pull request by lei-tang**
Add the monitoring of the "alpha.istio.io/canonical-serviceaccounts" annotation

### CODE REVIEW

File: security/pkg/registry/kube/service.go [hunk 1]
```diff
@@ -31,10 +31,10 @@ import (
 )
 
 // ServiceController monitors the service definition changes in a namespace. If a
-// new service is added with "alpha.istio.io/kubernetes-serviceaccounts" annotation
-// enabled, the corresponding service account will be added to the identity registry
+// new service is added with "alpha.istio.io/kubernetes-serviceaccounts" and
```
Comment (wattli) [inline security/pkg/registry/kube/service.go]: s/and/or, the two service accounts are logical 'or', i.e., customers should only specify one of them.

Comment (lei-tang) [reply security/pkg/registry/kube/service.go]: Done.

**Pull request merged**